PrestaShop/ps_imageslider
# Use AdminController

| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Migrate deprecated ajax_ps_imageslider.php into a controller to comply with https://github.com/PrestaShop/PrestaShop/issues/34578
| Type?             | improvement
| BC breaks?        | yes
| Deprecations?     | 
| Fixed ticket?     | 
| Sponsor company   | PrestaShop
| How to test?      | See below

## BC Breaks

File ajax_ps_imageslider.php has been removed

## How to test?

1. Go to back-office on PrestaShop 9 (`develop` branch) and click on "Configure" button of ps_imageslider module in Module Manager
2. Use the back-office page to drag-and-drop slides, and update their position. It should work as expected

This PR modified the function that perform the drag-and-drop of the slides, and store the updated positions.
